src/pages/Account.vue

```diff
@@ -7,38 +7,43 @@
 
                 <transition name="fade"><div v-if="errorMessage" v-bind:class="[goodFeedback ? 'good message' : 'message' ]">{{ errorMessage }}</div></transition>
 
+                <div v-if="voucherPayments.length > 0">
```

Comment:
@gingerbreadpup I thought we already had the function defined - but ah ha to that's for vouchers length